spec/spec_helper.rb

```diff
@@ -1,8 +1,19 @@
 if ENV['COVERAGE']
   require 'simplecov'
   SimpleCov.start 'rails' do
```

Comment:
ooo this reminds me should we add the RSpec XML formatting output too?

IDP example: https://github.com/18F/identity-idp/pull/7291

Reply:
added in 21b228d